File: Java/T4/proyectos/ProyectoEquipos/app/src/main/java/com/borja/proyectoequipos/utils/Jugador.java
package com.borja.proyectoequipos.utils;

import androidx.annotation.NonNull;

public class Jugador {

    String nombre, posicion;
    boolean estrella;

    public Jugador(String nombre, String posicion, boolean estrella) {
        this.nombre = nombre;
        this.posicion = posicion;
        this.estrella = estrella;
    }

    public String getNombre() {
        return nombre;
    }

    public String getPosicion() {
        return posicion;
    }

    public boolean isEstrella() {
        return estrella;
    }

    @NonNull
    @Override
    public String toString() {
        return getNombre();
    }
}

File: Kotlin/T1/codigos_ces/ExamenFantasy/src/model/Jugador.kt
package model

class Jugador(
    var id: Int? = null,
    var nombre: String? = null,
    var posicion: String? = null,
    var valor: Int? = null,
    var puntos: Int? = null
) {

    fun mostrarDatos() {
        println("id = ${id}")
        println("nombre = ${nombre}")
        println("posicion = ${posicion}")
        println("valor = ${valor}")
    }
}